### ISSUE TITLE
Headphone gestures/media control keys don't work

### ISSUE THREAD

**Issue by Movisivi** [Taiko2k/TauonMusicBox#780]
The application is working flawlessly the way it is except it is missing one crucial function. Can't skip track, go back to previous track or pause and resume playback using my bluetooth headphone or my keyboard trackpad  combo Logitech K400r. I am using Sony WH1000XM3 and running Windows 11 22H2 25145.1000

**Comment by Movisivi** [comment]
Do you have an update to give, if any?

**Comment by Movisivi** [comment]
@Taiko2k I just saw your recent commits and new releases. Thanks for improving the Windows version so quickly.

**Comment by Movisivi** [comment]
@Taiko2k just a heads up, the gestures don't work in latest Beta or Dev build of Windows 11. Reopening cause Beta is gonna be coming to stable soon

**Comment by Taiko2k** [comment]
> @Taiko2k just a heads up, the gestures don't work in latest Beta or Dev build of Windows 11. Reopening cause 22H2 is gonna be coming to stable soon

They didn't work in Tauon previously did they?

**Comment by Movisivi** [comment]
> They didn't work in Tauon previously did they?

Yes they did when you released the first Windows build (7.3.2). After that it stopped working. Okay I narrowed it down a bit:
So I did a fresh install of latest Dev (25174) yesterday and got all the updates and gestures work on first boot until I restarted. Same thing happened on latest Beta (22622).

**Comment by Movisivi** [comment]
Can confirm gestures work again after a fresh install of 25179 across multiple reboots. Weird. Guess it was a Windows issue? Unsure if I keep this open

**Comment by Movisivi** [comment]
Broken on 22621.382 after 7.4.0 update

**Comment by Taiko2k** [comment]
Tauon update didn't change anything though did it

**Comment by Movisivi** [comment]
Just gonna fault Windows here

**Comment by Movisivi** [comment]
@Taiko2k something noteworthy: If Tauon is already open when I connect my headphones then gestures won't work. If it is closed before I connect the headphones, gestures work.

**Comment by Movisivi** [comment]
Update from 7.4.7-2 to 7.5.0 kills the headphone gestures on my Bose QC45. Not even the workaround above works. Downgrading to 7.4.7 makes it work again.

**Comment by Taiko2k** [comment]
Yeah, from 7.5.0 Tauon no longer uses media key monitoring on Win 10/11 and instead now uses SMTC. I assumed this should fix the bluetooth issues, not make them worse.

**Comment by Movisivi** [comment]
Does 7.6.0 revert/fix this or should I stay on 7.4.7?

**Comment by Taiko2k** [comment]
No change with v7.6.0

**Comment by Movisivi** [comment]
Can you please post an update here when you get around to fixing this? I'm still using 7.4.7-2. Thanks in advance.

**Comment by Taiko2k** [comment]
There was a related fix in a recent release, try test on the latest version

**Comment by Movisivi** [comment]
Tried 7.6.6-1 and unfortunately no change. Headphone gestures and keyboard media controls are not working

**Comment by Taiko2k** [comment]
When Tauon is playing do you see Tauon in the Windows media desktop control thing? (The controls that come up when you click on the Windows status/clock area in the taskbar)

**Comment by Movisivi** [comment]
I don't see it on Windows 10 LTSC
